packages/client/src/declarations/index.ts

```diff
@@ -264,11 +265,15 @@ type RelationForCertificateCorrection =
   | 'CHILD'
 
 export type ICertificate = {
-  collector?: Partial<{ type: Relation | string }>
+  collector?: Partial<{
+    type: Relation | string
+    certificateTemplateId?: string
+  }>
   corrector?: Partial<{ type: RelationForCertificateCorrection | string }>
   hasShowedVerifiedDocument?: boolean
   payments?: Payment
-  data?: string
+  certificateTemplateId?: string
+  templateConfig?: ICertificateConfigData
```

Comment:
The id is enough for the template config to be fetched from the list of certificates, right? Is there a reason we need to store it temporarily and cannot fetch from the list when needed?

Reply:
I needed this because in the `query/registration-mapping`, I replace certificateTemplateId with templateConfig, did the vice versa in `mutation/registration-mapping`. Since we agreed to move the `templateConfig` from the certificate in the above conversation, this will be fixed too!